internal/DELETE.ME

```diff
@@ -0,0 +1,2 @@
+// DELETE ME
+// added to trigger docker build for all services
```

Comment:
let's not forget about this :D

Reply:
yeah i am just waiting for containers to spin up once in order to test if everything is fine with them. I'll revert the commit after  necessary checks have been made